release/downloadDubbo.sh

```diff
@@ -25,7 +25,7 @@ fi
 
 # Determine the latest Dubbo version by version number ignoring alpha, beta, and rc versions.
 if [ "${DUBBO_VERSION}" = "" ] ; then
-  DUBBO_VERSION="$(curl -sL https://github.com/dawnzzz/dubbo-kubernetes/releases | \
+  DUBBO_VERSION="$(curl -sL https://github.com/apache/dubbo/releases | \
```

Comment:
The name of this script is downloadDubbo.sh. This is how downloadDuboctl.sh is configured.

In other words, we only need one script to complete the work of downloading dubbo, dubboctl, and dubbo-cp, instead of two scripts.

Reply:
Thx reply😄

please check dubbo-release workflow in https://github.com/apache/dubbo-kubernetes/blob/master/.github/workflows/dubbo-release.yaml, we defined a versioned dubbo release consists of dubboctl + dubbo-cp.

Reply:
I think I understand it, I will file a pr correction, tks @dawnzzz